docs/database.rst

```diff
@@ -419,8 +419,8 @@ Create the test database from a custom SQL script
 
 You can replace the :fixture:`django_db_setup` fixture and run any code in its
 place. This includes creating your database by hand by running a SQL script
-directly. This example shows how sqlite3's executescript method. In more a more
-general use cases you probably want to load the SQL statements from a file or
+directly. This example shows how sqlite3's executescript method. In a more
```

Comment:
> This example shows how sqlite3's executescript method.

Missing a verb?

Reply:
Oh, nice catch. I believe that **"how"** is extra actually.

Reply:
Yes, please do.. - but a new commit is fine, too - will squash-merge it then.
Consider using `[ci skip]` in the message.

`checkqa` job currently fails due to newer flake8 it seems, so just ignore it.
Thanks!

Reply:
I've amended the commit and added `[ci skip]`. Thanks to you guys for the project, keep up the good work, glad I could help somehow 😄